homeassistant/components/syncthru/sensor.py

```diff
@@ -8,100 +8,107 @@
 import logging
 import voluptuous as vol
 
-from homeassistant.const import (
-    CONF_RESOURCE, CONF_HOST, CONF_NAME, CONF_MONITORED_CONDITIONS)
+from homeassistant.const import (CONF_RESOURCE, CONF_HOST, CONF_NAME,
+                                 CONF_MONITORED_CONDITIONS)
+from homeassistant.helpers import aiohttp_client
 from homeassistant.helpers.entity import Entity
 import homeassistant.helpers.config_validation as cv
 from homeassistant.components.sensor import PLATFORM_SCHEMA
 
-REQUIREMENTS = ['pysyncthru==0.3.1']
+REQUIREMENTS = ['pysyncthru==0.4.2']
 
 _LOGGER = logging.getLogger(__name__)
 
 DEFAULT_NAME = 'Samsung Printer'
-DEFAULT_MONITORED_CONDITIONS = [
-    'toner_black',
-    'toner_cyan',
-    'toner_magenta',
-    'toner_yellow',
-    'drum_black',
-    'drum_cyan',
-    'drum_magenta',
-    'drum_yellow',
-    'tray_1',
-    'tray_2',
-    'tray_3',
-    'tray_4',
-    'tray_5',
-    'output_tray_0',
-    'output_tray_1',
-    'output_tray_2',
-    'output_tray_3',
-    'output_tray_4',
-    'output_tray_5',
-]
-COLORS = [
-    'black',
-    'cyan',
-    'magenta',
-    'yellow'
-]
+COLORS = ['black', 'cyan', 'magenta', 'yellow']
+DRUM_COLORS = COLORS
+TONER_COLORS = COLORS
+TRAYS = range(1, 6)
+OUTPUT_TRAYS = range(0, 6)
+DEFAULT_MONITORED_CONDITIONS = []
+DEFAULT_MONITORED_CONDITIONS.extend(
+    ['toner_{}'.format(key) for key in TONER_COLORS])
+DEFAULT_MONITORED_CONDITIONS.extend(
+    ['drum_{}'.format(key) for key in DRUM_COLORS])
+DEFAULT_MONITORED_CONDITIONS.extend(['trays_{}'.format(key) for key in TRAYS])
+DEFAULT_MONITORED_CONDITIONS.extend(
+    ['output_trays_{}'.format(key) for key in OUTPUT_TRAYS])
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_RESOURCE): cv.url,
+    vol.Required(CONF_RESOURCE):
+    cv.url,
```

Comment:
Please revert all pure formatting changes. We shouldn't mix that into a fix PR.

Reply:
removed